### PR TITLE
Add Japanese translation index.mdx

### LINKED ISSUE #18413: Tranformers documentation translation to Japanese 🇯🇵
Hi!

Let's bring the documentation to all the Japanese-speaking community :)

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any, and we'll add your name to the list.

Some notes:

- Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗).
- Please translate in a gender-neutral way.
- Add your translations to the folder called `ja` inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
- Register your translation in `ja/_toctree.yml`; please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
- Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @omarespejel and @sgugger for review.
- 🙋 If you'd like others to help you with the translation, you can also post in the 🤗 [forums](https://discuss.huggingface.co/).

## Get Started section
- [x] [index.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/index.mdx) | https://github.com/huggingface/transformers/pull/21186
- [ ] [quicktour.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/quicktour.mdx).
- [x] [installation.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/installation.mdx) | https://github.com/huggingface/transformers/pull/21241

## Tutorial section
- [ ] [pipeline_tutorial.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/pipeline_tutorial.mdx) 
- [ ] [autoclass_tutorial.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/autoclass_tutorial.mdx) 
- [ ] [preprocessing.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/preprocessing.mdx) 
- [ ] [training.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/training.mdx) 
- [ ] [accelerate.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/accelerate.mdx) 
- [ ] [model_sharing.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_sharing.mdx) 
- [x] [multilingual.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/multilingual.mdx) | https://github.com/huggingface/transformers/pull/21084

### PR DESCRIPTION
**Pull request by kambehmw**
# What does this PR do?

Adds Japanese translation to index.mdx

Fixes #18413 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@omarespejel @sgugger

### CODE REVIEW

**Comment by sgugger** [comment]
@ArthurZucker could you have a look?

File: docs/source/ja/index.mdx [hunk 1]
```diff
@@ -0,0 +1,392 @@
+<!--Copyright 2020 The HuggingFace Team. All rights reserved.
```
Comment (ArthurZucker) [inline docs/source/ja/index.mdx:1]:
```suggestion
<!--Copyright 2023 The HuggingFace Team. All rights reserved.
```

File: docs/source/ja/index.mdx [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+1. **[AltCLIP](model_doc/altclip)** (from BAAI) released with the paper [AltCLIP: Altering the Language Encoder in CLIP for Extended Language Capabilities](https://arxiv.org/abs/2211.06679) by Chen, Zhongzhi and Liu, Guang and Zhang, Bo-Wen and Ye, Fulong and Yang, Qinghong and Wu, Ledell.
+1. **[Audio Spectrogram Transformer](model_doc/audio-spectrogram-transformer)** (from MIT) released with the paper [AST: Audio Spectrogram Transformer](https://arxiv.org/abs/2104.01778) by Yuan Gong, Yu-An Chung, James Glass.
+1. **[BART](model_doc/bart)** (from Facebook) released with the paper [BART: Denoising Sequence-to-Sequence Pre-training for Natural Language Generation, Translation, and Comprehension](https://arxiv.org/abs/1910.13461) by Mike Lewis, Yinhan Liu, Naman Goyal, Marjan Ghazvininejad, Abdelrahman Mohamed, Omer Levy, Ves Stoyanov and Luke Zettlemoyer.
```
Comment (ArthurZucker) [inline docs/source/ja/index.mdx:55]: For all of these I think we have the Japanese template now that should also automatically update in Japanese. You can probably copytaste from the main `README.mdx`

**Review by ArthurZucker — COMMENTED** [review]
Let's just change the models list to Japanese! l

File: docs/source/ja/index.mdx (hunk 1 above)
Comment (kambehmw) [reply docs/source/ja/index.mdx]: Thank you for the review!
I fixed the year of the license from 2020 to 2023.

File: docs/source/ja/index.mdx (hunk 2 above)
Comment (kambehmw) [reply docs/source/ja/index.mdx]: I copy-pasted from `README_ja.md` and changed the list of models to Japanese.
https://github.com/huggingface/transformers/blob/main/README_ja.md

**Review by ArthurZucker — APPROVED** [review]

**Comment by ArthurZucker** [comment]
Thanks a lot for this 🚀

**Pull request merged**